src/Sessions.h

```diff
@@ -212,12 +216,16 @@ class NetSessions {
 	bool CheckHeaderTrunc(int proto, uint32_t len, uint32_t caplen,
 			      const Packet *pkt, const EncapsulationStack* encap);
 
+	void InsertConnection(ConnectionMap* m, const ConnIDKey& key, Connection* conn);
```

Comment:
Might helpful to document "must not already be in the map" as a pre-condition for this function.